pnpm-lock.yaml

```diff
@@ -400,22 +400,26 @@ importers:
       '@next/eslint-plugin-next': 12.3.2-canary.18
```

Comment:
The lock file should not be changed by examples, please revert the changes in this file.